src/transformState.js

```diff
@@ -6,6 +6,25 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const action of actions) {
+    switch (true) {
+      case action.type === 'addProperties':
```

Comment:
```suggestion
    switch (action.type) {
      case 'addProperties':
```

Reply:
please, read the documentation and fix all cases
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch